lib/omniauth/strategies/twitch.rb

```diff
@@ -58,7 +58,7 @@ def authorize_params
           options[:authorize_options].each do |k|
             params[k] = request.params[k.to_s] unless [nil, ''].include?(request.params[k.to_s])
           end
-
+          params[:redirect_uri] = options[:redirect_uri] if options[:redirect_uri] != nil
```

Comment:
Line is too long. [89/80]<br>Prefer `!expression.nil?` over `expression != nil`.